src/libAtomVM/term.h

```diff
@@ -1330,14 +1330,14 @@ static inline int term_exactly_equals(term a, term b, Context *ctx)
 }
 
 /**
- * @brief Returns 1 if given terms are equal.
+ * @brief Returns \c true if given terms are equal.
  *
- * @details Compares 2 given terms and returns 1 if they are the same or they have same numeric value.
+ * @details Compares 2 given terms and returns true if they are the same or they have same numeric value.
  * @param a first term
  * @param b second term
- * @return 1 if they are the same, 0 otherwise.
+ * @return \c true if they are the same, \c false otherwise.
  */
-static inline int term_equals(term a, term b, Context *ctx)
+static inline bool term_equals(term a, term b, Context *ctx)
 {
     if (a == b) {
         return 1;
```

Comment:
Identically, we are missing true here and it could be a single line or expression.